extra/release_tool.py

```diff
@@ -816,13 +814,17 @@ def check_tag_availability(state):
     tag_avail = {}
     for repo in Component.get_components_of_type("git"):
         tag_avail[repo.git()] = {}
+        missing_repos = False
         try:
             execute_git(state, repo.git(), ["rev-parse", state[repo.git()]['version']],
                         capture=True, capture_stderr=True)
             # No exception happened during above call: This is a final release
             # tag.
             tag_avail[repo.git()]['already_released'] = True
             tag_avail[repo.git()]['build_tag'] = state[repo.git()]['version']
+        except FileNotFoundError as err:
+            print(err)
+            missing_repos = True
```

Comment:
Are you sure it produces a `FileNotFoundError`, and not a similar error, but from Git? I haven't checked, so you may be right. I just wanna make sure.

Reply:
What it fails is the `cd` into the directory.

If the directory exists but it is not a git repo, then I guess Git will fail with different error and then the exception will be raised, right?

Reply:
Ah, I see. Yes, you're right.